docs/source/guide/get_started/api_tutorial.rst

```diff
@@ -4,68 +4,63 @@
 Besides CLI functionality, The OpenVINO™ Training Extension provides APIs that help developers to integrate OpenVINO™ Training Extensions models into their projects.
 This tutorial intends to show how to create a dataset, model and use all of the CLI functionality through APIs.
 
-For demonstration purposes we will use the Object Detection ATSS model with `WGISD <https://github.com/thsant/wgisd>`_ public dataset as we did for the :doc:`CLI tutorial <../tutorials/base/how_to_train/detection>`.
+OTX aims to provide a unified interface for interacting with different backends, enabling seamless training and validation across both native and third-party backends. This allows users to easily adapt and integrate popular computer vision models.
+OpenVINO™ Training Extensions APIs are designed to be easy to use and flexible, allowing you to create, train, and deploy deep learning models with minimal effort.
+
+There are :doc:`dedicated tutorials <../tutorials/base/how_to_train/index>` in our documentation with life-practical examples on specific datasets for each task.
 
 .. note::
 
     To start with we need to `install OpenVINO™ Training Extensions <https://github.com/open-edge-platform/training_extensions/blob/develop/QUICK_START_GUIDE.md#setup-openvino-training-extensions>`_.
 
-*******************
-Dataset preparation
-*******************
-
-1. Clone a repository
-with `WGISD dataset <https://github.com/thsant/wgisd>`_.
-
-.. code-block:: shell
-
-    cd data
-    git clone https://github.com/thsant/wgisd.git
-    cd wgisd
-    git checkout 6910edc5ae3aae8c20062941b1641821f0c30127
```

Comment:
I think including a real-world example in the API tutorial is quite valuable, users could immediately see results and provides a good starting point for experimentation and visualisation. Could you clarify the reason for removing it?

Reply:
I’d prefer to keep using WGISD here, mainly because the API tutorial is meant to be a quick 10-minute guide. And since OTX primarily supports COCO format, using a dataset in another format might cause confusion or errors for users unfamiliar with this requirement. WGISD being in COCO format helps reduce this friction and makes it easier for users to follow along without worrying about dataset compatibility.

Also, I don't think duplication in the documentation is necessarily a bad thing as I think providing key information in multiple places can ensure users don’t miss it.

Reply:
NP, will return the WGISD example